### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fishyvishy**
fixed User Stories with Code links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ The Fridge and User entities use the Factory design pattern in the FridgeFactory
 ![UML Diagram](reports/UMLDiagram.png)
 
 ## User Stories With Code
-- [Register Use Case] (https://github.com/CSC207-2022F-UofT/course-project-recipez/blob/main/Reports/RegisterUsecase.jpeg)
-- [Enter ingredient Use case] (https://github.com/CSC207-2022F-UofT/course-project-recipez/blob/main/Reports/EnterIngredientsUsecase.jpeg)
-- [Search Use Case] (https://github.com/CSC207-2022F-UofT/course-project-recipez/blob/main/Reports/SearchUseCase.png)
+- [Register Use Case] (https://github.com/CSC207-2022F-UofT/course-project-recipez/blob/main/reports/RegisterUsecase.jpeg)
+- [Enter ingredient Use case] (https://github.com/CSC207-2022F-UofT/course-project-recipez/blob/main/reports/EnterIngredientsUsecase.jpeg)
+- [Search Use Case] (https://github.com/CSC207-2022F-UofT/course-project-recipez/blob/main/reports/SearchUseCase.png)
```
Comment (Shaffaan) [inline README.md:40]: Can you add a screenshot of test coverage in the readme, it is already there in the reports folder

**Comment by fonuxxine** [comment]
Looks good!

**Review by fonuxxine — APPROVED** [review]
Looks good!

**Review by Shaffaan — APPROVED** [review]

**Review by EricKarpovits — APPROVED** [review]
Looks good vish

**Pull request merged**